### PR TITLE
additional minor fixes

### PR DESCRIPTION
**Pull request by NullableInt**
Fixes several small issues in the code

### CODE REVIEW

File: _includes/payment-link.md [hunk 1]
```diff
@@ -72,9 +72,10 @@ the consumer when the payment has been fulfilled.
 
 ### API Requests
 
-The API requests depend on the payment instrument you are using when implementing
-the Payment Link scenario, see [purchase flow][purchase-flow] One-phase payment
-instruments will not implement `capture`, `cancellation` or `reversal`.
+The API requests depend on the payment instrument you are using when
+implementing the Payment Link scenario, see [purchase flow][purchase-flow]
```
Comment (asbjornu) [inline _includes/payment-link.md]:
```suggestion
implementing the Payment Link scenario, see [purchase flow][purchase-flow].
```

File: _includes/payment-link.md [hunk 2]
```diff
@@ -72,9 +72,10 @@ the consumer when the payment has been fulfilled.
...
+One-phase payment instruments will not implement `capture`, `cancellation` or
+`reversal`.
 The options you can choose from when creating a payment with key `operation`
 set to Value `Purchase` are listed below.
```
Comment (asbjornu) [inline _includes/payment-link.md]:
```suggestion
set to `Purchase` are listed below.
```

File: _includes/payment-link.md [hunk 3]
```diff
@@ -72,9 +72,10 @@ the consumer when the payment has been fulfilled.
...
 ### API Requests
 
-The API requests depend on the payment instrument you are using when implementing
-the Payment Link scenario, see [purchase flow][purchase-flow] One-phase payment
-instruments will not implement `capture`, `cancellation` or `reversal`.
+The API requests depend on the payment instrument you are using when
+implementing the Payment Link scenario, see [purchase flow][purchase-flow]
+One-phase payment instruments will not implement `capture`, `cancellation` or
+`reversal`.
 The options you can choose from when creating a payment with key `operation`
```
Comment (asbjornu) [inline _includes/payment-link.md]:
```suggestion
The options you can choose from when creating a payment with `operation`
```

File: _includes/settlement-reconciliation.md [hunk 4]
```diff
@@ -127,22 +127,22 @@ transaction list. Below is a sequence diagram detailing the interaction.
...
-deactivate PayEx.BackOffice
-
-activate Merchant
-
-PayEx.BackOffice->>Merchant: Balance Report (PDF-file)
-note left of Merchant: files are sent by e-mail or file transfer
-PayEx.BackOffice->>Merchant: Transaction list (XLSX-file)
-PayEx.BackOffice->>Merchant: Transaction list (XML-file)
+    activate Merchant
+    activate PayEx.FrontEnd
```
Comment (asbjornu) [inline _includes/settlement-reconciliation.md]: 1. `PayEx` should be replaced with `SwedbankPay`
2. I don't think we need to separate between `FrontEnd` and `BackOffice`.
3. When using programmatic names with dots, underscores, CamelCase, etc., in diagrams, please provide an alias with `participant SwedbankPay as Swedbank Pay` in the start of the diagram.

File: _includes/callback-reference.md [hunk 5]
```diff
@@ -72,9 +73,10 @@ about this update.
 {% endif %}
 
 {:.table .table-striped}
-| **Parameter** | **Description**
-| `Payment Instrument` | `CreditCard`, `Invoice`, `Swish`, `Vipps`, `DirectDebit`, `MobilePay`
-| `Transaction Type` | Authorizations, Captures, Cancellations, Reversals
+|    **Parameter**     | **Description**                                                       |
```
Comment (asbjornu) [inline _includes/callback-reference.md]:
```suggestion
|    Parameter     | Description                                                       |
```

File: _includes/callback-reference.md [hunk 6]
```diff
@@ -59,11 +60,11 @@ about this update.
 ```js
 {
     "payment": {
-        "id": "/psp/<payment instrument>/payments/<payment-id>",
+        "id": "/psp/{{ payment-instrument }}/payments/<payment-id>",
```
Comment (asbjornu) [inline _includes/callback-reference.md]:
```suggestion
        "id": "/psp/{{ payment-instrument }}/payments/{{ page.paymentId }}",
```

File: _includes/callback-reference.md [hunk 7]
```diff
@@ -39,14 +40,14 @@ about this update.
...
+        "instrument": "{{ payment-instrument }}"
     },
     "payment": {
-        "id": "/psp/<payment instrument>/payments/<payment-id>",
+        "id": "/psp/{{ payment-instrument }}/payments/<payment-id>",
         "number": 222222222
     },
     "transaction": {
-        "id": "/psp/<payment instrument>/payments/<payment-id>/<transaction type>/<transaction-id>",
+        "id": "/psp/{{ payment-instrument }}/payments/<payment-id>/<transaction type>/<transaction-id>",
```
Comment (asbjornu) [inline _includes/callback-reference.md]:
```suggestion
        "id": "/psp/{{ payment-instrument }}/payments/{{ page.paymentId }}/<transaction type>/{{ page.transactionId }}",
```

File: _includes/callback-reference.md [hunk 8]
```diff
@@ -72,9 +73,10 @@ about this update.
 {% endif %}
 
 {:.table .table-striped}
-| **Parameter** | **Description**
-| `Payment Instrument` | `CreditCard`, `Invoice`, `Swish`, `Vipps`, `DirectDebit`, `MobilePay`
-| `Transaction Type` | Authorizations, Captures, Cancellations, Reversals
+|    **Parameter**     | **Description**                                                       |
+| :------------------: | :-------------------------------------------------------------------- |
+| `Payment Instrument` | `CreditCard`, `Invoice`, `Swish`, `Vipps`, `DirectDebit`, `MobilePay` |
```
Comment (asbjornu) [inline _includes/callback-reference.md]:
```suggestion
```

File: _includes/callback-reference.md [hunk 9]
```diff
@@ -72,9 +73,10 @@ about this update.
...
+|  `Transaction Type`  | Authorizations, Captures, Cancellations, Reversals                    |
```
Comment (asbjornu) [inline _includes/callback-reference.md]:
```suggestion
| `<transaction type>` | `authorizations`, `captures`, `cancellations`, `reversals`
```

File: _includes/callback-reference.md [hunk 10]
```diff
@@ -59,11 +60,11 @@ about this update.
 ```js
 {
     "payment": {
-        "id": "/psp/<payment instrument>/payments/<payment-id>",
+        "id": "/psp/{{ payment-instrument }}/payments/<payment-id>",
         "number": 222222222
     },
     "transaction": {
-        "id": "/psp/<payment instrument>/payments/<payment-id>/<transaction type>/<transaction-id>",
+        "id": "/psp/{{ payment-instrument }}/payments/<payment-id>/<transaction type>/<transaction-id>",
```
Comment (asbjornu) [inline _includes/callback-reference.md]:
```suggestion
        "id": "/psp/{{ payment-instrument }}/payments/{{ page.paymentId }}/<transaction type>/{{ page.transactionId }}",
```

File: _includes/callback-reference.md [hunk 11]
```diff
@@ -39,14 +40,14 @@ about this update.
 {
     "paymentOrder": {
         "id": "/psp/paymentorders/<paymentorder-id>",
-        "instrument": "<payment instrument>"
+        "instrument": "{{ payment-instrument }}"
     },
     "payment": {
-        "id": "/psp/<payment instrument>/payments/<payment-id>",
+        "id": "/psp/{{ payment-instrument }}/payments/<payment-id>",
```
Comment (asbjornu) [inline _includes/callback-reference.md]:
```suggestion
        "id": "/psp/{{ payment-instrument }}/payments/{{ page.paymentId }}",
```

File: _includes/settlement-reconciliation.md (hunk 4 above)
Comment (NullableInt) [reply _includes/settlement-reconciliation.md]: Updated

**Pull request merged**